packages/style-engine/src/vite-config.ts

```diff
@@ -0,0 +1,9 @@
+import { vanillaExtractPlugin } from "@vanilla-extract/vite-plugin";
+
+export const styleEngineViteConfig = {
+  plugins: [
+    vanillaExtractPlugin({
+      identifiers: ({ hash }) => `tgph_${hash}`,
```

Comment:
We have this and `tgph-${hash}` in [style-engine-config.ts](https://github.com/knocklabs/telegraph/pull/161/files#diff-5ee0bad7e3cbdea091c27630187ea010fc5d17bba93a9d99f64a5585338bf2c9)

Should these be aligned?

Reply:
Oh I just need to remove this from `vite-config.ts`